dbt_semantic_interfaces/protocols/meta.py

```diff
@@ -0,0 +1,12 @@
+from abc import abstractmethod
+from typing import Any, Dict, Protocol
+
+
+class ConfigMeta(Protocol):  # noqa: D
```

Comment:
I think it would be better to stick with our existing naming conventions to avoid confusion. (I got confused reading this 😅 ) So then this protocol would be called `SemanticLayerElementConfig`, and the pydantic implementation becomes `PydanticSemanticLayerElementConfig`.

Reply:
Good callout!